docs/guides/cli.md

```diff
@@ -24,9 +24,11 @@ Other options include:
   --quiet, -q                  no logging - output only                                     [boolean]
 ```
 
-> Note: The Spectral CLI supports both YAML and JSON.
+The Spectral CLI supports loading documents as YAML or JSON, and validation of OpenAPI v2/v3 documents via our built-in ruleset. 
 
-Currently, Spectral CLI supports validation of OpenAPI v2/v3 documents via our built-in ruleset, or you can create [custom rulesets](../getting-started/rulesets.md) to work with any JSON/YAML documents.
+Custom rulesets can be provided with `--ruleset`, but the default place Spectral CLI looks for a ruleset is in the current working directory for a file called `.spectral.yml`.
```

Comment:
Suggest somehow indicating it doesn't have to be `.spectral.yml` (can be `.spectral.json`). Something like the below?

```
You can also provide your own ruleset file. By default, the Spectral CLI will look for a ruleset file called `.spectral.yml` or `.spectral.json` in the current working directory. You can tell spectral to use a different file by using the `--ruleset` CLI option.
```

Reply:
🤔 merged, so I guess the thought was this is not important?

Reply:
It was a lovely suggestion and I made it. I didn’t check the commit had pushed up properly and merged, it’s sat on my local and will just go up with whatever PR I make next.